userguide/directoryservice.rst

```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
-   | User Base                | string        | ✓        | distinguished name (DN) of the user container in Active Directory                                                             |
+   | User Base                | string        | ✓        | Distinguished name (DN) of the user container in Active Directory.                                                            |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Group Base               | string        | ✓        | distinguished name (DN) of the group container in Active Directory                                                            |
+   | Group Base               | string        | ✓        | Distinguished name (DN) of the group container in Active Directory.                                                           |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Site Name                | string        | ✓        | the relative distinguished name of the site object in Active Directory                                                        |
+   | Site Name                | string        | ✓        | The relative distinguished name of the site object in Active Directory.                                                       |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Domain Controller        | string        | ✓        | will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
-   |                          |               |          | specified, %brand% selects the closest DC which responds; use the short form of the FQDN                                      |
+   | Domain Controller        | string        | ✓        | Will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
```

Comment:
Normally, change "will be" to "is", but in this case it's not needed at all: "Automatically added to..."